docs/installation/ubuntulinux.md

```diff
@@ -357,6 +357,8 @@ To specify a DNS server for use by Docker:
 
         $ sudo nano /etc/default/docker
 
+>**Note**:For  Ubuntu 15.04, place a drop-in file in   `/etc/systemd/system/docker.service.d` and modify that with the properties below instead of working on the `/etc/default/docker` file.
```

Comment:
Maybe we can prevent using a "note", by adding something before line 352, e.g.:

```Markdown
The instructions below describe how to configure a daemon running on
Ubuntu 14.10 or below. Ubuntu 15.04 and above uses systemd as its boot
and service manager. Refer to [control and configure Docker with systemd](../articles/systemd.md#custom-docker-daemon-options)
to configure a daemon controled by systemd.
```

wdyt?